### DISCUSSION TITLE
HyperlinkButton Design

### DISCUSSION THREAD

**Discussion by YanLi2024** [BenjaminKey/Tafe24S1T105_Ext#4] [General]
Hello Ben
I am working on the buttons at this moment. I set up a HyperlinkButton for CurrencyConverter page and another one for the MainMenu page (Currency Converter Button). When I click the these buttons, the pages can jump to the destination pages successfully. But there is a problem. When I hover the cursor to these HyperlinkButton, the backgound color disappears, but the text is still there. I think you need to design your button too. Wondering how you did yours?

**Comment by BenjaminKey** [comment]
Hi Yan

I have not set any hover attributes to the buttons, so I am not sure why that has happen
I did have to rename my buttons with a 2 to avoid conflicts with buttons on the main menu.
I have copied my button stackpanel below.

hope this helps let me know if you cant get it to work

<StackPanel Width="250">
				<TextBlock x:Name="titleTextBlock" Width="200" TextAlignment="Center" Foreground="#FF2B1295" FontWeight="Bold" Text="Universal Calculator" FontSize="20" Margin="10"></TextBlock>
				<Button x:Name="math2CalculatorButton" Width="150" Margin="0,0,0,10" Content="Math Calculator" HorizontalAlignment="Center" Click="math2CalculatorButton_Click" ></Button>
				<Button x:Name="mortgageCalculator2Button" Width="150" Margin="0,0,0,10" Content="Mortgage Calculator" HorizontalAlignment="Center" Click="mortgageCalculator2Button_Click" ></Button>
				<Button x:Name="currencyConverter2Button" Width="150" Margin="0,0,0,10" Content="Currency Converter" HorizontalAlignment="Center" Click="currencyConverter2Button_Click"></Button>
				<Button x:Name="exit2Button" Width="150" Margin="0,0,0,10" Content="Exit" HorizontalAlignment="Center" Click="exit2Button_Click" ></Button>
</StackPanel>

**Comment by YanLi2024** [comment]
Hi Ben
Thanks for your help. I tried to set up click event for Exit button on Currency Converter page, but it didn't work in the first place. So I used hyperlinkButton. I saw you used buttons instead of hyperlinkButton, I will use button too to keep consistency with your design. 

one small suggestion, I think you could consider removing Mortgage Calculator and Exit buttons below the Universal Calculator heading.
